app/src/main/java/cm/aptoide/pt/home/BundlesResponseMapper.java

```diff
@@ -41,11 +44,31 @@
 
           appBundles.add(new AppBundle(widget.getTitle(), applicationsToApps(
               ((ListApps) widget.getViewObject()).getDataList()
-                  .getList(), type), type, event, widget.getTag()));
+                  .getList(), type, widget.getTag()), type, event, widget.getTag()));
         } else if (type.equals(HomeBundle.BundleType.ADS)) {
-          appBundles.add(
-              new AdBundle(widget.getTitle(), ((GetAdsResponse) widget.getViewObject()).getAds(),
-                  new Event().setName(Event.Name.getAds), widget.getTag()));
+          appBundles.add(new AdBundle(widget.getTitle(),
+              new AdsTagWrapper(((GetAdsResponse) widget.getViewObject()).getAds(),
+                  widget.getTag()), new Event().setName(Event.Name.getAds), widget.getTag()));
+        } else if (type.equals(HomeBundle.BundleType.SOCIAL)) {
+          List<Card> list = ((SocialResponse) widget.getViewObject()).getDataList()
+              .getList();
+          if (!list.isEmpty()) {
+            List<App> apps = new ArrayList<>();
+            Card card = list.get(0);
+            App app = card.getApp();
+            apps.add(app);
+            if (card.hasUser()) {
+              appBundles.add(
+                  new SocialBundle(applicationsToApps(apps, type, widget.getTag()), type, event,
+                      widget.getTag(), card.getUser()
+                      .getAvatar(), card.getUser()
+                      .getName()));
+            } else {
+              appBundles.add(
+                  new SocialBundle(applicationsToApps(apps, type, widget.getTag()), type, event,
+                      widget.getTag(), R.mipmap.ic_launcher, marketName));
```

Comment:
widget.getTag, widget.getViewObject and widget.getTitle is being called a lot, maybe it could be set in the start of the for.